src/app/globals.css

```
@@ -457,3 +457,19 @@ header {
+
+/* ─────────────────────────────────────────────
+   ACCESSIBILITY & SEO UTILITY
+   ───────────────────────────────────────────── */
+.sr-only {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)